src/core/server/ui_settings/settings/date_formats.ts

```diff
@@ -170,7 +170,7 @@ export const getDateFormatSettings = (): Record<string, UiSettingsParams> => {
           dateNanosLink:
             // TODO: [RENAMEME] Need prod urls.
             // issue: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/335#issuecomment-868294864
-            '<a href="https://opensearch.org/docs/opensearch/units" target="_blank" rel="noopener noreferrer">' +
+            '<a href="https://opensearch.org/docs/latest/opensearch/units/" target="_blank" rel="noopener noreferrer">' +
```

Comment:
Can we use the `DocLinksService` to get a versioned link here? Same thing for all the changes below as well.

Reply:
I ran into some issues when try use `DocLinksService` in all `src/*/server/*` modules. For example:
- `src/core/server/ui_settings/settings/date_formats.ts`
- `src/plugins/data/server/ui_settings.ts`
- `src/plugins/map_lagacy/server/ui_settings.ts`
I think the issue is due to a circular dependency situation here: 
I need the `DocLinksService` instance (from `opensearch_dashboards_react/public`)to generate url links in `server/ui_setting.ts`, but `ui_setting.ts` is required for plugin server services setup / start (https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/data/server/plugin.ts#L109). I might be wrong with the root cause but I don't have a good way to inject the `DocLinksService` yet.